.github/workflows/sync-contribution-base-branch.yml

```diff
@@ -0,0 +1,29 @@
+name: Sync contributor base branch
+on:
+  schedule:
+    # runs at minute 0 of the 0th and 12th hour every day.
+    - cron:  '0 0,12 * * *'
+
+jobs:
+  sync_contributor_base_branch:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Setup Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: '3.8'
+      - name: Install Python Dependencies
+        run: |
+          python -m pip install --upgrade pip
+          pip install pipenv
+      - name: Sync Base Branch to Master
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
should this be `CONTENTBOT_GH_ADMIN_TOKEN` ?

Reply:
Good catch. My bad.